### PR TITLE
[ROCm] Script updates for the ROCm CSB build

### PR DESCRIPTION
**Pull request by deven-amd**
This PR contains commits that add / update scripts required to setup the ROCm Community Supported Build (CSB).

@gunan , please review the `tensorflow/tools/ci_build/linux/rocm/run_csb_tests.sh`. It is based on your recommendation of the bazel test command to use, and will be the script we use for our CSB.

------------------------------------------------

@tatianashp @whchung @parallelo @gunan

### CODE REVIEW

**Review by whchung — APPROVED** [review]

File: tensorflow/tools/ci_build/linux/rocm/run_csb_tests.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+echo ""
+
+# Run configure.
+export PYTHON_BIN_PATH=`which python3`
+export CC_OPT_FLAGS='-mavx'
+
+export TF_NEED_ROCM=1
+export TF_GPU_COUNT=${N_GPUS}
+
+yes "" | $PYTHON_BIN_PATH configure.py
```
Comment (gunan) [inline tensorflow/tools/ci_build/linux/rocm/run_csb_tests.sh]: JFYI, this is not testing XLA at all.
in case you also want to test XLA.

Comment (deven-amd) [reply tensorflow/tools/ci_build/linux/rocm/run_csb_tests.sh]: >JFYI, this is not testing XLA at all.

that is intentional. ROCm support for XLA is still in the process of being upstreamed. 

XLA unit tests will not pass until the upstreaming of ROCm support is complete, and hence leaving those out for now. Will enable them once the upstreaming process is complete.

**Comment by deven-amd** [comment]
@gunan, gentle ping.   please approve PR. thanks

File: tensorflow/tools/ci_build/builds/pip.sh [hunk 2]
```diff
@@ -131,15 +131,19 @@ echo "Using Bazel flags: ${BAZEL_FLAGS}"
 
 PIP_BUILD_TARGET="//tensorflow/tools/pip_package:build_pip_package"
```
Comment (gunan) [inline tensorflow/tools/ci_build/builds/pip.sh]: @av8ramit are we using this script?

for rocm, we may want to remove this script from the critical path.

**Review by gunan — APPROVED** [review]

**Comment by deven-amd** [comment]
The errors in "Ubuntu Sanity" CI run, do not seem to be related to any changes in this PR. 

Let me know if there is anything that needs to be done on my end to resolve them. 

thanks

**Pull request merged**